crouton.php

```diff
@@ -5,7 +5,7 @@
 Description: A tabbed based display for showing meeting information.
 Author: bmlt-enabled
 Author URI: https://bmlt.app
-Version: 3.16.3
+Version: 3.17.1
```

Comment:
shouldn't this just be 3.17.0 instead of 3.17.1, as their was no 3.17.0 release

Reply:
What ever happened to PHP6, or Java4?  During development, 3.17.0 was the embedded map and OSM integration, 3.17.1 was improvements to Meeting-Details-Page generation and other UI elements.  It's just that 3.17.0 never made it to release, because development of 3.17.1 sort of overtook it.  It doesn't matter, really, but I would like to keep them a an unreleased version and a release, if only to keep the change list for each manageable.

Reply:
that's fine I mainly don't like that there is no tag associated with the changelog but I guess 3.17.0 beta3 would cover it? Make sure to bump the readme.txt to 3.17.1 as well if thats what you want to use